docs/DeveloperGuide.md

```diff
@@ -87,7 +87,7 @@ The following sequence diagram shows the relevant interactions behind `setPriori
 
 #### Study Plan Feature
 
-##### Proposed Implementation
+#### Proposed Implementation
```

Comment:
I changed it because in the actual website, the Proposed Implementation headers are of an even smaller font than the main text. Not sure if there's a better way to resolve this :O

Reply:
Maybe we could merge `Proposed Implementation` with `Study Plan Feature` and have `Study Plan Feature - Proposed Implementation` with 4 #'s? `Proposed Implementation` is the only subsection under `Study Plan Feature` anyway haha

Reply:
Updated all the headers according to your suggestion!
I also added the confirmation message to setPriorityLevel seq diag to keep it consistent with yours.

Reply:
> Updated all the headers according to your suggestion!
> I also added the confirmation message to setPriorityLevel seq diag to keep it consistent with yours.

Nice!